src/com/operators_loops/MultiplicationTable.java

```diff
@@ -3,7 +3,8 @@
 public class MultiplicationTable {
     public static void main(String[] args) {
         for (int i = 0; i < 10; i++) {
-            System.out.printf("%5d", ((i+1)*4));
+            int r = (4*(i+1));
+            System.out.println("4 * " + (i+1) + " = " + r);
```

Comment:
теж формат поправив. Пробіли ставляться біля математичних операторів для кращої читаємості 
```suggestion
            int r = (4 * (i + 1));
            System.out.println("4 * " + (i + 1) + " = " + r);
```